### PR TITLE
chore(nimbus): Update docs hub URL and add ADR for domain

### PR DESCRIPTION
**Pull request by k88hudson**
See also https://github.com/mozilla/experimenter-docs/pull/36, which needs to be merged first

Because

* We are updating docs hub URL to a custom domain

This commit

* Updates the link on the front page to point to experimenter.info
* Adds an ADR describing the decision

### CODE REVIEW

**Review by jaredlockhart — COMMENTED** [review]
LGTM but https://experimenter.info/ still shows no pages?

**Comment by k88hudson** [comment]
Blocked on mozilla/experimenter-docs#36 merging

**Review by jodyheavener — APPROVED** [review]
Yeah LGTM once the domain is correctly set up

**Comment by k88hudson** [comment]
Yeah we have to do both at once or else it will show up as an error (since the path changes from /experimenter-docs to /)

**Pull request merged**
